money.go

```diff
@@ -38,6 +39,12 @@ func WithRoundingMode(mode string) MoneyOption {
 	}
 }
 
+func WithShowZero(show bool) MoneyOption {
```

Comment:
we cannot change the show zero option afterward in this case?

Reply:
setShowZero may be need to call this function first

Reply:
i mean after creating a money instance with showZero = false, there's no way to change it to true?